lib/models/ObjectMDAmzRestore.ts

```diff
@@ -11,6 +11,7 @@
 export default class ObjectMDAmzRestore {
     'expiry-date': Date | string;
     'ongoing-request': boolean;
+    'content-md5': string;
```

Comment:
Can we add getter/setter functions (like other fields), and cover it with unit tests in `tests/unit/models/ObjectMDAmzRestore.spec.js`?

Reply:
Not worth the effort, this type is actually not used at all... at the moment, I would be more inclined to remove these accessors, which are akin to dead code

The only point of adding these here is for documentation, to ensure the fields can indeed be discovered by looking at these ˋmodels`

Reply:
So I guess we can take advantage of the PR to remove these functions?

Reply:
we could ; but best do this separately: we have a real fix (on mongodb), and just adding the field for documentation.
Cleanup will be best done separately, after thinking through how we want to evolve the models (to make them more useful): remove unneeded fields, always use the models, ...

Reply:
we could ; but best do this separately: we have a real fix (on mongodb), and just adding the field for documentation.
Cleanup will be best done separately, after thinking through how we want to evolve the models (to make them more useful): remove unneeded fields, always use the models, ...

Reply:
I don't mind but this change is already unrelated with the PR objective (fix of the listing), so it would be logical to either follow the existing pattern, or do it separately as well?

Reply:
ticket for followup : https://scality.atlassian.net/browse/ARSN-434

(i can remove the field, but the goal was really only to keep up with the reality, and ensure we don't use a field which is not even mentioned in the models)